server.go

```diff
@@ -329,7 +329,10 @@ func (s *server) AddHandlerRoute(handler ChannelHandler, method string, action s
 	method = strings.ToLower(method)
 	channelType := strings.ToLower(string(handler.ChannelType()))
 
-	path := fmt.Sprintf("/%s/{uuid:[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}}/%s", channelType, action)
+	path := fmt.Sprintf("/%s/{uuid:[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}}", channelType)
```

Comment:
The callback URL confifured looks like `/c/jc/abcd-abcd-abcd-abcd/` 

Here I make sure we support those for `""` action passed to the AddHandlerRoute